Add Tbb and Tcl files (#396)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/tcl.md

---
name: Tcl
category: Languages and Frameworks
description: Tcl is a versatile programming language and framework that enables the development of custom software applications by integrating diverse applications, protocols, devices, and frameworks.
download_url: https://github.com/tcltk/tcl/tags
works_on_arm: true
supported_minimum_version:
    version_number: 8.5.12
    release_date: 2012/07/25

optional_info:
    homepage_url: https://www.tcl-lang.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://github.com/tcltk/tcl/tree/main/unix#readme
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Linux/ARM64 release notes are not available. Installation and Testing were done using released tar files.

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. Installation and testing are done using the released tar archives.
```
